tests/vmipreset_test.go

```diff
@@ -137,7 +137,7 @@ var _ = Describe("[rfe_id:609][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 			Expect(err).ToNot(HaveOccurred())
 
 			Expect(reviewResponse.Details.Causes).To(HaveLen(1))
-			Expect(reviewResponse.Details.Causes[0].Field).To(Equal("spec.domain.devices.disks[1]"))
+			Expect(reviewResponse.Details.Causes[0].Field).To(Equal("spec.domain.devices.disks[2]"))
```

Comment:
Why do we need to change the disk index?

Reply:
Hi @alicefr  it is because of this change. https://github.com/kubevirt/kubevirt/pull/12516/files#diff-1ffe3878238ae2b0809d647ed00f3f2db70c6ff5d96b181dda8c08f907566949R83-R86

The Alpine already has two disks one is disk0 and cloudinit disk which is added during the creation due to the above change.  So, the test disk we are adding to the preset is at the location disks[2]